docs/InstallGuide.md

```diff
@@ -0,0 +1,143 @@
+</tr>
+<tr>
+<td> torq </td>
+<td>
+
+
+Is a mandatory parameter that is the full path or relative path to the TorQ installation. It can either be a TorQ Directory where the version is already unzipped, that can be used when multiple TorQ Applications are used on the server for example and all point to a single TorQ main code. This will create a softlink to the relevant TorQ code. Or it can be a .tar.gz file of the TorQ installation for a fresh install. 
+Example usage in the script:
+
+`torq=/home/user/TorQ/TorQ-3.7.0`
+
+Where TorQ-3.7.0 is unzipped directory from the latest release .tar.gz file. Or
+
+`torq=/home/user/TorQ/TorQ-3.7.0.tar.gz` 
+
+Which is the .tar.gz file from GitHub using:
+
+`wget --content-disposition https://github.com/AquaQAnalytics/TorQ/archive/3.7.0.tar.gz`
+
+
+</td>
+</tr>
+<tr>
+<td> releasedir </td>
+<td>
+
+Is a mandatory parameter that is the full path or relative path to the deployment directory that will populate the TorQ and TorQApp. 
+If the directory doesn't exist then script creates one. It can be anything, 
+as per previous AquaQ instructions the folder name has been deploy. 
+The releasedir parameter can be used as follows:
+
+`releasedir=/home/user/deploy`
+
+</td>
+<tr>
+<td> installfile </td>
+<td>
+
+Is a mandatory parameter with the full path or relative path to the TorQApp installation file (ACCEPTS ONLY .tar.gz FILE). 
+Can be used as follows:
+
+`installfile=/home/user/TorQ-FSP/TorQ-Finance-Starter-Pack-master.tar.gz`
+
+</td>
+</tr>
+<tr>
+<td> data </td>
+<td>
+
+An optional parameter. That is if you want to have your database live in a different part of the system rather than the place where the code lives. Can be used as follows:
+
+`data=/home/data/torq_data`
+
+If the directory doesn't exist the script will make one. Also accepts a relative path if necessary. 
+
+</td>
+</tr>
+<tr>
+<td> env </td>
+<td>
+
+Env is the environment-specific optional installation parameters. That is a separate .sh script that can be configured for different environments like DEV/UAT/PROD. In the script, there are SED replacements for necessary variables. If this parameter is left empty or isn't included nothing happens. If you want to include it you have to insert the parameters as follows (also accepts relative path):
+
+`env=/home/user/env_spec_dev.sh` 
+
+`env=env_spec_dev.sh`
+
+Below is a user guide on how to set up the .sh script to have necessary replacements by the env parameter.
+For env parameter the env_spec script should look like this:
+
+`echo $1`
+
+`find $1 -type f -name "*.sh"`
+
+`find $1 -type f -name "*.sh" -exec sed -i "s/export KDBBASEPORT=.*/export KDBBASEPORT=7373/g" {} \;`
+
+Create an sh script env_spec_dev.sh and then add the parameter to the install script start line. 
+
+`env=/home/user/env_spec_dev.sh`
+
+This will change the KDBBASEPORT to a new value.
+Similar actions can be done with other variables.
+But requires user basic knowledege of sed commands. 
```

Comment:
Similar actions can be done with other variables, and required user basic knowledge of sed commands